cms/envs/common.py

```diff
@@ -1264,7 +1264,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
+HELP_TOKENS_INI_FILE = REPO_ROOT / "cms" / "envs" / "help_tokens.ini"
```

Comment:
@robrap Now that the files are no longer in a "docs" directory, the new filenames do make it more clear what the files are for.